tests/js/garbage-collection.js

```diff
@@ -0,0 +1,62 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2016 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+'use strict';
+
+const Realm = require('realm');
+const TestCase = require('./asserts');
+
+const NUMBER_OF_OBJECTS = 1000;
+const BUFFER_LENGTH = 1024;
+const READ_CYCLES = 10;
+
+module.exports = {
```

Comment:
Perhaps it would make sense with a brief comment about why we have this? Including a link to the issue in the Electron repo